uwblueprint/community-fridge-kw
# feat: Create custom radio select component

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* Created RadioSelectButton which is grouped into RadioSelectGroup 
* See Date and Time select as example to see how the radio select group component is used



## Steps to test

https://user-images.githubusercontent.com/37950626/142348372-3355335e-ca23-4a5d-8ec3-c2b30ed9ff6c.mov

1. Go to scheduling process's Select Date and Time page and select a daypart. It should update the formValues (you can test using console.log in SelectDateAndTime component)
2. When you go 'Back' or 'Next' and have selected an option, it should remain selected when returning to this page.


## What should reviewers focus on?
* 


## Checklist
